docs/dependency-specification.md

```diff
@@ -180,6 +180,16 @@ with the corresponding `add` call:
 poetry add https://example.com/my-package-0.1.0.tar.gz
 ```
 
+## Dependency `extras`
+
+You can specify [PEP-508 Extras](https://www.python.org/dev/peps/pep-0508/#extras)
+for a dependency:
+
+```toml
+[tool.poetry.dependencies]
+gunicorn = { version = "^20.1", extras = ["gevent"] }
+```
+
```

Comment:
```suggestion
{{% note %}}
These activate extra defined for the dependency, to configure an optional dependency for extras in your project refer to <link>.
{{% /note %}}
```

Maybe something like this will be beneficial to distinguish between extra activation and definition?

Reply:
Added